src/main/kotlin/no/nav/syfo/client/oppfolgingstilfelle/OppfolgingstilfellePersonDTO.kt

```diff
@@ -31,3 +33,18 @@ fun OppfolgingstilfellePersonDTO.toLatestOppfolgingstilfelle(): Oppfolgingstilfe
         )
     }
 }
+
+fun List<OppfolgingstilfelleDTO>.toOppfolgingstilfelle(): List<Oppfolgingstilfelle> {
+    return this.map {
+        Oppfolgingstilfelle(
+            start = it.start,
+            end = it.end
+        )
+    }
+}
+
+fun List<Oppfolgingstilfelle>.findOppfolgingstilfelleByDate(date: LocalDate): Oppfolgingstilfelle? {
+    return this.find {
+        it.start.isBeforeOrEqual(date) && it.end.isAfterOrEqual(date)
+    }
+}
```

Comment:
Jeg tror det er mulig å få flere tilfeller som overlapper med en gitt dato, så man burde kanskje ta et valg for hva man skal gjøre da?

Reply:
Good catch! Legger til logikk for det.